cql3/query_processor.cc

```diff
@@ -538,6 +538,9 @@ future<::shared_ptr<cql_transport::messages::result_message>> query_processor::e
     size_t retries = remote_.get().mm.get_concurrent_ddl_retries();
     while (true)  {
         try {
+            if (statement->needs_topology_quiesce()) {
+                co_await remote_.get().ss.await_topology_quiesced();
```

Comment:
you're right.
next version will deal with that by holding on to the table's effective replication map before waiting for topology not-busy state, to prevent new migrations from starting.

Reply:
gone in v2